src/detect-threshold.c

```diff
@@ -372,6 +379,43 @@ static int ThresholdTestParse05(void)
     return 0;
 }
 
+/**
+ * \test ThresholdTestParse06 is a test for thresholding by_both
+ *
+ *  \retval 1 on succces
```

Comment:
nit: s/cuces/success/ (here, and a few more places).

Reply:
Will fix.